src/matlab_tools/euclidean_to_wheel.m

```diff
@@ -0,0 +1,56 @@
+function target_wheel_speeds = euclidean_to_wheel(current_wheel_speeds, target_euclidean_velocity)
```

Comment:
nit: maybe put a note up top here and say this is a test script that can be used to check the real thing

you can also move this file to `src/physics`, i don't think we will have any more matlab tools in this folder

Reply:
do you mean create a new folder `src/physics` or move it to `src/software/physics`

Reply:
ah my bad, I mean't `src/software/physsics` same folder as the other class